user/api.go

```diff
@@ -176,197 +206,213 @@ func (a *Api) CreateChildUser(res http.ResponseWriter, req *http.Request) {
 // status: 500 STATUS_ERR_UPDATING_USR
 func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[string]string) {
 
-	if td := getUnpackedToken(req.Header.Get(TP_SESSION_TOKEN), a.ApiConfig.Secret); td != nil {
```

Comment:
We check `td != nil`, but we do not in the updated code.  Will td ever be nil if err is nil?  Or is checking the err enough?